src/main/java/controllers/PlayerCreationInformation.java

```diff
@@ -0,0 +1,35 @@
+package controllers;
+
+/**
+ * A class containing the direct instructions of a User regarding a Player in a Game's creation.
+ */
+public class PlayerCreationInformation {
+    private String playerName;
```

Comment:
Again consider using name and isComp for clarity. Same with the constructor, the parameters are being assigned to their respective attributes with the same variable names.